spec/features/user_spec.rb

```diff
@@ -75,4 +75,19 @@
       expect(Devise.mailer.deliveries.first.to).to include(user.email)
     end
   end
+
+  scenario 'new users must be members' do
+    user = User.create(
```

Comment:
I'd advise to use `create!` instead of `create`. How come? Because if the user creation process fails, it will be harder to debug - the `create` method will just silently fail while the `create!` method will immediately show what the error is!

Reply:
@DamirSvrtan @m-pereira  this feels like a unit/model test, I don't see anything integration specific here, wdyt?

Reply:
@DamirSvrtan nice! doing it now!

Reply:
@dixpac Yeah it would be good, however I can't see the users' role anywhere in UI, how are you thinking that would be?

Reply:
Currently you are using `feature` test which is slow and doesn't test anything UI specific in your case 😄 , you are just testing when new user is created that default role is `member`. I'm thinking that this test needs to be movied into `spec/models/user_spec.rb`

Reply:
I'm sorry :satisfied:

Reply:
Thanks for catching that one @dixpac and thanks for updating the PR @m-pereira! Merging it in!